buildtools/cli.go

```diff
@@ -566,9 +566,18 @@ func NugetCmd(c *cli.Context) error {
 		return err
 	}
 
+	allowInsecureConnection, err := extractBoolFlagFromArgs(&filteredNugetArgs, "allow-insecure-connections")
```

Comment:
why we need to add this flag to cli?
isn't it a nuget's flag?

Reply:
It is a NuGet flag, but I wanted to add the ability for the users to control the flag.
Do you suggest something else?